src/components/List/List.js

```diff
@@ -1,14 +1,17 @@
-import React, { useState, useEffect } from 'react';
+import React from 'react';
 import { firestore } from '../../lib/firebase.js';
 import { useCollectionData } from 'react-firebase-hooks/firestore';
 import GroceryItem from '../GroceryItem/GroceryItem';
 import NavBar from '../NavBar/NavBar';
 import { useHistory } from 'react-router-dom';
 
 const List = ({ token, setToken }) => {
-  const listData = firestore.collection('groceryItems'); // decided to connect with the 'groceryitems' Collection to see if it will render and successfully it does. Ultimately we want to get the 'Items' Collection inside of 'Lists'
-  const query = listData.orderBy('createdAt');
-  const [groceryItems] = useCollectionData(query, { idField: 'id' });
+  const [listData] = useCollectionData(
+    firestore //changed the 'listData' variable to the entire database to render different collections instead of the one "groceryItems" collection
+      .collection(localStorage.getItem('token'))
```

Comment:
Since we are passing the token in from props, you could just replace localStorage.getItem('token') with token

Reply:
@jandrade @vanmars I would have to experiment with this today. If I'm stuck, I may need assistance with applying this method to the code and will reach out. But thanks for you both. Great reviewing!

Reply:
👋 please don't mind tackling this right now if you feel that it will take some extra time. This can be added later on 😃.

📖  **Documentation-tip:** You can add TODO comments in the code, so you can give notes to your future self (or other folks) about something that can be added/updated/impoved, so y'll can make some minor (non-blocking) changes on future PRs.

e.g.
```js
// TODO: Change localStorage.token to use the token prop instead.
const [token, setToken] = useState(window.localStorage.getItem('token')); 
```

Reply:
Oh and one more thing... no need to add that `// TODO` comment here, but wanted to share this tip with y'all that has been really helpful when I work on projects :).

Reply:
Thanks Juan, noted and will do!!